src/definition/accessors/IUserRead.ts

```diff
@@ -8,4 +8,9 @@ export interface IUserRead {
     getById(id: string): Promise<IUser>;
 
     getByUsername(username: string): Promise<IUser>;
+
+    /**
+     * Gets the app user of this app.
+     */
+    getAppUser(): Promise<IUser>;
```

Comment:
this should follow the signature from the method in the `IUserBridge` interface https://github.com/RocketChat/Rocket.Chat.Apps-engine/blob/master/src/server/bridges/IUserBridge.ts#L8